<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceRegister</class>
 <widget class="QWidget" name="DeviceRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nightwood - Register Device</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f8f9fa;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>24</number>
   </property>
   <property name="leftMargin">
    <number>40</number>
   </property>
   <property name="topMargin">
    <number>36</number>
   </property>
   <property name="rightMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>36</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 36px; font-weight: bold; color: #2d3436; letter-spacing: 1.2px; padding-bottom: 8px;</string>
     </property>
     <property name="text">
      <string>Register Device</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="minimumHeight">
      <number>48</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="subtitleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 15px; color: #636e72; margin-bottom: 8px;</string>
     </property>
     <property name="text">
      <string>Scan the QR code with your device or enter the codes manually</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="qrCodeLayout">
     <item>
      <spacer name="hSpacerLeft">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="qrCodeLabel">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>240</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white; border: 1.5px solid #dfe6e9; border-radius: 16px;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacerRight">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="authCodeBoxLayout">
     <item>
      <layout class="QHBoxLayout" name="codeRow1">
       <item>
        <widget class="QLineEdit" name="codeEdit1">
         <property name="readOnly"><bool>true</bool></property>
         <property name="alignment"><set>Qt::AlignmentFlag::AlignCenter</set></property>
         <property name="maxLength"><number>11</number></property>
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #f5f6fa; border-radius: 8px; border: 1.5px solid #dfe6e9; padding: 8px; color: #222; letter-spacing: 2px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="codeEdit2">
         <property name="readOnly"><bool>true</bool></property>
         <property name="alignment"><set>Qt::AlignmentFlag::AlignCenter</set></property>
         <property name="maxLength"><number>11</number></property>
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #f5f6fa; border-radius: 8px; border: 1.5px solid #dfe6e9; padding: 8px; color: #222; letter-spacing: 2px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="codeRow2">
       <item>
        <widget class="QLineEdit" name="codeEdit3">
         <property name="readOnly"><bool>true</bool></property>
         <property name="alignment"><set>Qt::AlignmentFlag::AlignCenter</set></property>
         <property name="maxLength"><number>11</number></property>
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #f5f6fa; border-radius: 8px; border: 1.5px solid #dfe6e9; padding: 8px; color: #222; letter-spacing: 2px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="codeEdit4">
         <property name="readOnly"><bool>true</bool></property>
         <property name="alignment"><set>Qt::AlignmentFlag::AlignCenter</set></property>
         <property name="maxLength"><number>11</number></property>
         <property name="font">
          <font>
           <family>Courier New</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #f5f6fa; border-radius: 8px; border: 1.5px solid #dfe6e9; padding: 8px; color: #222; letter-spacing: 2px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="copyButton">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  font-size: 16px;
  font-weight: bold;
  color: #6c5ce7;
  background-color: white;
  border-radius: 10px;
  border: 2px solid #6c5ce7;
}
QPushButton:hover {
  background-color: #f5f3ff;
}
QPushButton:pressed {
  background-color: #eeeaff;
}</string>
       </property>
       <property name="text">
        <string>Copy All Codes</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>54</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  font-size: 16px;
  font-weight: bold;
  color: #6c5ce7;
  background-color: white;
  border-radius: 10px;
  border: 2px solid #6c5ce7;
}
QPushButton:hover {
  background-color: #f5f3ff;
}
QPushButton:pressed {
  background-color: #eeeaff;
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 